Sources/Model/ObjC/Common/TracksDeviceInformation.m

```diff
@@ -206,6 +206,51 @@ - (UIInterfaceOrientation)statusBarOrientation {
 }
 #endif
 
+/// Preferred reading content size based on the accessibility setting of the iOS device.
+/// 
+/// - This will be null for Mac OS.
+///
+- (NSString *)preferredContentSizeCategory {
+#if TARGET_OS_IPHONE
+    if ([NSThread isMainThread]) {
+        return UIApplication.sharedIfAvailable.preferredContentSizeCategory;
+    }
+
+    __block NSString *preferredContentSizeCategory;
+
+    dispatch_sync(dispatch_get_main_queue(), ^{
+        preferredContentSizeCategory = UIApplication.sharedIfAvailable.preferredContentSizeCategory;
```

Comment:
It has been a long time for me as well. 😅

> do you know if this pauses the return statement below until this line is called when it's not on the main thread?

Yes, as we dispatching synchronously using `dispatch_sync`, this will wait before executing the following `return` statement.